data/macros.meson

```diff
@@ -45,3 +45,9 @@
         --num-processes %{_smp_build_ncpus} \
         --print-errorlogs \
         %{nil}}
+
+# Macros for declarative buildsystems of rpm >= 4.20
```

Comment:
s/of/if?

Reply:
> s/of/if?

I think **declarative buildsystem of rpm >= 4.20** means declarative buildsystems is one of the feature of rpm >= 4.20.